pages/linux/pacdiff.md

```diff
@@ -0,0 +1,29 @@
+# pacdiff
+
+> Pacorig, pacnew and pacsave maintenance utility for pacman.
+> If you want to use a different program for diffs, set the `DIFFPROG` enviromental variable to point to your desired binary, with any needed flags. `vim -d` is the default.
```

Comment:
The description section should be ["short and snappy"](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout) per the style guide - the intent is to convey what a tool does any why someone might want to use it. Controlling the diff program is technical information; if we want to include that at all then it should probably be an example, not part of the description.

Is it very common to change the DIFFPROG command, or is there a reason to highlight this behavior early on?

Reply:
Unless you know how to use vim for diffs (I certainly don't) `pacdiff` is a really difficult tool to use :laughing:. I'll add an example instead.

Reply:
The example looks great, thanks!